### ISSUE TITLE
Not working on Elitebook 8770w

### ISSUE THREAD

**Issue by faisal330** [chris1111/HP-Probook-EliteBook-Package-Creator#30]
Hi Chris,
I have been following your guide to install Hackintosh on my Hp Elitebook 8770w. After creating usb when I try to boot it got stuck at the following error:

![WhatsApp Image 2020-03-25 at 2 30 18 PM](https://user-images.githubusercontent.com/49313531/77521911-366d1880-6ea5-11ea-8335-6dcb324e7e3c.jpeg)

I did a little bit of searching about it and I found out it is related to some kind of graphics issue. You know better then me as I'm still a noob. I also tried some online solutions like: (Inject Nvidia, Inject Web Driver, nv_disable=1 bla bla...) but noting works. It would be great if you can fix this issue. because there are no specific guides for my laptop as compare to other probooks, elitebooks.

Thanks!

My Laptop Specs:
Core i7 3720qm 2.6ghz
8gb ddr3 ram
Nvidia quadro K3000m(No intel graphics)
1920x1080 display
samsung 250gb ssd

**Comment by chris1111** [comment]
You are wrong about Hardware/Graphics CPU
You have [Intel HD 4000](https://ark.intel.com/content/www/fr/fr/ark/products/64891/intel-core-i7-3720qm-processor-6m-cache-up-to-3-60-ghz.html)
See in Bios if you have an option to disable the Nvidia quadro K3000 ?
if not use boot args nv_disable=1 and Inject Intel= false
What is the bios version you have ?

**Comment by faisal330** [comment]
Yes I know my processor came with HD4000 but they are non-switchable I don't know why. The only working graphics is Nvidia and also even I can't install Intel drives in windows. It only shows Nvidia in device manager. Actually my model only support Nvidia. The Intel drivers are disabled by the manufacture. So that's why I said (No intel graphics)

![Annotation 2020-03-25 160405](https://user-images.githubusercontent.com/49313531/77529880-493a1a00-6eb2-11ea-9ba3-3e7f062e1288.png)

Here is my BIOS version : 

![12](https://user-images.githubusercontent.com/49313531/77530093-a2a24900-6eb2-11ea-81a8-bf64ce48e1e9.jpeg)

And there is no option to disable drivers in BIOS.

**Comment by faisal330** [comment]
I tried booting with nv_disable=1 and inject intel=false. It boot but the display resolution is so bad and video memory is like 3mb
![WhatsApp Image 2020-03-25 at 4 43 31 AM](https://user-images.githubusercontent.com/49313531/77533124-8d4a0200-6e53-11ea-832d-010fdf689630.jpeg)
![WhatsApp Image 2020-03-25 at 4 43 31 AM (1)](https://user-images.githubusercontent.com/49313531/77533147-93d87980-6e53-11ea-9163-be0c385e4e11.jpeg)

I also tried booting with only inject intel=false then the OS boot but after that there is only black screen.

**Comment by faisal330** [comment]
Previously I was following a Hackintosh Zone guide that kind a work for me, I got booted into the OS, the display is working fine, graphic acceleration is also working 

![Screenshot 2020-03-25 at 5 02 54 AM](https://user-images.githubusercontent.com/49313531/77535405-cd12e880-6e57-11ea-8fb0-c92f98817ce7.png)

But all other things are not working like (battery status, brightness controls, keyboard, trackpad, audio). That's why I'm trying to use your guide

I attached the EFI folder of Hackintosh Zone Guide, In case you want to have a look.

Thanks!
